app/src/main/res/layout/activity_main.xml

```diff
@@ -13,12 +13,12 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
-            android:layout_marginTop="8dp"
-            android:layout_marginEnd="8dp"
+            android:layout_marginTop="@dimen/default_margin"
+            android:layout_marginEnd="@dimen/default_margin"
             android:linksClickable="false"
             android:textAlignment="center"
             android:textColor="#FF0000"
```

Comment:
Colors should be centralised as well, please add them to one file and use references instead